driver/src/main/java/org/neo4j/driver/v1/exceptions/NotCommittedException.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2002-2016 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.driver.v1.exceptions;
+
+/**
+ * An <em>NotCommittedException</em> indicates that a unit of work could not successfully be committed.
+ * @since 1.1
+ */
+public class NotCommittedException extends Neo4jException
```

Comment:
We should document the fact that there are cases where it could have committed, but we just were unable to find out that it committed...